Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -4661,7 +4661,7 @@ public void Actions_PressingAndReleasingButtonInSameUpdate_StillTriggersAction()
             ctx => { ++receivedCalls; };
         action.Enable();
 
-        var firstState = new GamepadState {buttons = 1 << (int)GamepadButton.B};
```

Comment:
The fix shouldn't have any impact on existing tests?

Reply:
Hence I wouldn't expect any changes to any tests.

Reply:
I believe this is due to my first pass where I removed the aliased values. Will revert